### PR TITLE
Update Ontarian -> Canadian 

### LINKED ISSUE #180: Replace current tagline on homepage
"Amplifying the stories of the pandemic throughout Canada"

### PR DESCRIPTION
**Pull request by silent1mezzo**
fixes #180

### CODE REVIEW

**Comment by silent1mezzo** [comment]
#### 📝Changed routes:


| [**/about**](https://www-1p9yfye7j-my-covid-story.vercel.app/about) | [**/faq**](https://www-1p9yfye7j-my-covid-story.vercel.app/faq) |
|:-:|:-:|
|<a href="https://www-1p9yfye7j-my-covid-story.vercel.app/about">Screenshot of /about</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HXW-OAqIuoyA4p_Vz5MyNehAVRZ0XqW3NQMpMT6x_FdJsAuey0hHTx5Kj_iM-QAQuEmdTbwu3yJL4cKT9hgp3g.png">(view full size)</a> |<a href="https://www-1p9yfye7j-my-covid-story.vercel.app/faq">Screenshot of /faq</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/1F0IWPe7iLKb5v48BF7GmmF8xMjrjwNHyhs0qNb6oAtDD2rRMR7HmaASokUaGs_Hmgy6LeUKsg4UQH9omepXjg.png">(view full size)</a> |



| [**/new**](https://www-1p9yfye7j-my-covid-story.vercel.app/new) |
|:-:|
|<a href="https://www-1p9yfye7j-my-covid-story.vercel.app/new">Screenshot of /new</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/uKNBMv4yhjyk7MTN0MVdkj0iHAk99WTM34hrYi8w8T7KTrnuZAlQ_JcAuU9B3YyTB2StCoCb4RjVjc1qIsacXA.png">(view full size)</a> |







Commit 5a588339dc52f777918fe5ec22f1cc4f5c96a7f7 (https://www-1p9yfye7j-my-covid-story.vercel.app).

**Review by davesteinberg — CHANGES_REQUESTED** [review]
I have to question most of the FAQ edits. Much of the original language was specifically about the leadership of the response in Ontario, and it's inaccurate to generalize that to the whole country. e.g. significant portions of the country were never in a humanitarian crisis, leadership across the country has ranged from excellent (in Atlantic Canada) to mixed (Quebec and BC) to disastrous (Ontario, Manitoba, and Alberta). And some of the original language might not even apply to Ontario anymore (I don't think we're being led _into_ a humanitarian crisis anymore).

I think we should get some input from the comms folks before updating this content.

File: pages/about.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function About() {
             Who We Are
           </Heading>
           <Text>
-            We are a group of concerned Ontarians who can no longer stand by as our province is led
+            We are a group of concerned Candians who can no longer stand by as our country is led
```
Comment (davesteinberg) [inline pages/about.tsx:21]: Missing an a in Canadians.

**Comment by davesteinberg** [comment]
The new copy is here: https://docs.google.com/document/d/1KA-oULnf45t5SL8dBhj8D2IJWmEE1oIR4C-ARyZlYZY/edit?usp=sharing

Also, Natasha highlighted the request from Kapil to remove his name from the list on the About page by the end of this week. Is that doable for you, @silent1mezzo?

**Pull request closed without merging**